docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -0,0 +1,389 @@
+---
+id: validator-deployment-guide
+title: Test Cases
+hide_title: true
+sidebar_label: Validator Deployment Guide
+slug: /mine-hnt/validators/validator-deployment-guide
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Validator Deployment Guide 
+
+This guide will cover the end-to-end process of deploying a Helium Validator. It assumes you're starting from scratch. At the end, you'll have a Helium Validator deployed the Network.
+
+:::info
+
+* We're currently operating the Testnet, so you'll be deploying to the Testnet until Mainnet is live. This guide reflects deploying to the Testnet. 
+* Use the #validator channel on the [Helium Discord server](https://discord.gg/helium) to take part in the real-time Validator development and get support.
+
+:::
+
+
+## Prerequisites
+
+To successfully deploy a Helium Validator, you'll need to do the following:
+
+* Install the [Helium CLI Wallet](https://github.com/helium/helium-wallet-rs)
```

Comment:
Based on a question in discord, it made me wonder whether people could assume this means running the wallet on the same machine as the validator. So either here or down in `Create Testnet Wallet` maybe clarify that? 

Or, decide that the target audience understands crypto wallets, dunno.